.travis.yml

```diff
@@ -15,5 +15,5 @@ before_script:
   - pip install --user vim-vint
   - vim --version
 script:
-  - make test
+  - make test_slow
```

Comment:
Slow/fast should only make a difference if the tests are hanging.

Reply:
It seems like it should, but now every test is failing. I only made that change in an attempt to debug. I plan on reverting it.

Reply:
Actually, I was planning on force-pushing the previous commit, not reverting. But now I'm not sure if that's a good idea.

Reply:
Both should work, we can easily squash-merge it later.